src/mouse/default_gutter_handler.js

```diff
@@ -105,14 +130,28 @@ function GutterHandler(mouseHandler) {
     function hideTooltip() {
         if (tooltipTimeout)
             tooltipTimeout = clearTimeout(tooltipTimeout);
-        if (tooltipAnnotation) {
+        if (tooltipContent) {
             tooltip.hide();
-            tooltipAnnotation = null;
+            tooltipContent = null;
             editor._signal("hideGutterTooltip", tooltip);
             editor.off("mousewheel", hideTooltip);
         }
     }
 
+    function annotationsToSummaryString(annotations) {
+        var isMoreThanOneAnnotationType = false;
+        var summaryString = "";
+        for (var i = 0; i < 3; i++){
+            var annotationType = ['error', 'warning', 'info'][i];
```

Comment:
we can simplify and get rid of magic number 3 here. we can use es6 since it's all over the codebase now
```js
const annotationTypes = ['error', 'warning', 'info'];
for (const annotationType of annotationTypes) {...}
```
and if `annotationTypes` are also used/available somewhere else, would be great to have them specified in a single place